pandas/core/frame.py

```diff
@@ -2005,25 +2009,25 @@ def to_dict(
 
         You can specify the return orientation.
 
-        >>> df.to_dict('series')
+        >>> df.to_dict(orient='series')
```

Comment:
We could also keep allowing just `orient` as positional only keyword, given how we have been using that ourselves in the docs?

Reply:
@jorisvandenbossche should I remove the keyword `orient` from the examples as well?